docker/llm/finetune/xpu/README.md

```diff
@@ -1,13 +1,13 @@
-## Fine-tune LLM with IPEX LLM Container
+## Finetune LLM with IPEX LLM Container
 
-The following shows how to fine-tune LLM with Quantization (QLoRA built on IPEX-LLM 4bit optimizations) in a docker environment, which is accelerated by Intel XPU.
+The following shows how to finetune LLM with Quantization (QLoRA built on IPEX-LLM 4bit optimizations) in a docker environment, which is accelerated by Intel XPU.
```

Comment:
Remove `(QLoRA built on IPEX-LLM 4bit optimizations)` here or make it more generic since this image supports various optimizations for QLoRA, LoRA, QALoRA, etc.

Reply:
Done. Added examples link in README.